src/experimental/reputation/system.ts

```diff
@@ -41,7 +43,14 @@ export const DEFAULT_AGREEMENT_WEIGHTS: ReputationWeights = {
  * Default reputation service URL.
  * @experimental
  */
-export const DEFAULT_REPUTATION_URL = "https://reputation.dev-test.golem.network/v1/providers/scores";
+export const DEFAULT_REPUTATION_URL = "https://reputation.dev-test.golem.network/v2/providers/scores";
+
+/**
+ * The number of top scoring providers to consider when selecting an agreement.
+ *
+ * Default for `topPoolSize` agreement selector option.
+ */
+export const DEFAULT_AGREEMENT_TOP_POOL_SIZE = 2;
```

Comment:
Just curious if that's not to small for the default?

Reply:
It is small, but also the pool of providers is not big. In my tests keeping this value low did affect the results by alot (10-20%).

Due to the limited amount of good proposals readily available, pool size of 5 had pretty much the same result as default random agreement selector.